### ISSUE TITLE
Do not include php dependencies in the repo

### ISSUE THREAD

**Issue by m1kola** [torchbox/design-in-browser-bootstrap#43]
Is that possible to not commit PHP dependencies into the repository? It seems to me that they can be ignored and can be installed on developer's machine via dependency managing tool (see [php composer](https://getcomposer.org/)).

**Comment by Loque-** [comment]
It is - it would add an additional step however as users would need to install them before using the bootstrap. Ideally php wouldn't be a dependency TBH.

Maybe adding a whitelist for patternlab; `!/patternlab/**` to .gitignore would be a suitable solution in the meantime?

**Comment by m1kola** [comment]
> it would add an additional step however as users would need to install them before using the bootstrap

Right, but we already ask users to install JS dependencies using `npm` / `yarn`, so shouldn't be a big problem, I think.

> Ideally php wouldn't be a dependency TBH.

That would be nice. It looks like there is a node version of patternlab: https://github.com/pattern-lab/patternlab-node

Probably same templates can be rendered using `twig.js`, but I'm not sure. I'm not familiar with this stack.

**Comment by Loque-** [comment]
Well, it is very easy to create a version of this without the php dependancies baked in, and the install steps for composer. 

The repository is originally aimed at use by designers who are the primary audience, and the response from other developers being requested to use yarn has not always been positive also (yet another thing to install).

I tried the node version before settling on this version based on php. A lot of hours went into that, but can't remember the specifics of why it was unfortunately unsuitable.

For the moment I am going to add the whitelist as a useful reference. If I get time I'll do an example without the php dependencies baked in!
